src/main/java/conan/actions/ActionUtils.java

```diff
@@ -51,6 +62,11 @@ static void runInstall(Project project, Component component, boolean update) {
      * @param update    true if it's update and install action.
      */
     private static void matchProfilesAndInstall(Project project, Component component, boolean update) {
+
+        if(!isConanInstalled(project)){
```

Comment:
White-space should be added after the "if" to be compatible with the existing code conventions.
Let's also remove the first new-line in the method.